extensions/api/api-core/src/main/java/org/eclipse/dataspaceconnector/api/query/QuerySpecDto.java

```diff
@@ -0,0 +1,128 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.api.query;
+
+import jakarta.validation.constraints.AssertTrue;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
+import jakarta.ws.rs.QueryParam;
+import org.eclipse.dataspaceconnector.spi.query.SortOrder;
+
+public class QuerySpecDto {
+
+    @QueryParam("offset")
+    @PositiveOrZero
+    private Integer offset = 0;
+
+    @QueryParam("limit")
+    @Positive
+    private Integer limit = 50;
+
+    @QueryParam("filter")
+    private String filter;
+
+    @QueryParam("sort")
+    private String sortOrder = "ASC";
```

Comment:
Is it possible to change the type to `SortOrder` instead of String? This seems to cause the allowed enum values to be removed from the openAPI yaml files.

Reply:
I used a string to decouple the `dto` model from the `domain` model. 
But yes, introducing a `SortOrderDto` seems like to much, I will set it back to `SortOrder`